edocumentation-application/src/main/java/org/dcsa/edocumentation/controller/BookingController.java

```diff
@@ -5,25 +5,38 @@
 import org.dcsa.edocumentation.transferobjects.BookingRefStatusTO;
 import org.dcsa.edocumentation.transferobjects.BookingTO;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 @Validated
 @RestController
 @RequiredArgsConstructor
 public class BookingController {
   private final BookingService bookingService;
 
-  @PostMapping(path = "${spring.application.bkg-context-path}" + "/bookings")
+  @PostMapping(path = "${spring.application.bkg-context-path}/bookings")
   @ResponseStatus(HttpStatus.CREATED)
   public BookingRefStatusTO createBooking(@Valid @RequestBody BookingTO bookingTO) {
     assert bookingTO.carrierBookingRequestReference() == null;
     return bookingService.createBooking(bookingTO);
   }
 
+  @GetMapping(
+      path = "${spring.application.bkg-context-path}/bookings/{carrierBookingRequestReference}",
+      produces = MediaType.APPLICATION_JSON_VALUE)
+  @ResponseStatus(HttpStatus.OK)
+  public ResponseEntity<BookingTO> getBooking(
+      @Valid @PathVariable("carrierBookingRequestReference") @NotNull @Size(max = 100)
+          String carrierBookingRequestReference) {
+    return bookingService
+        .getBooking(carrierBookingRequestReference)
+        .map(ResponseEntity::ok)
+        .orElse(ResponseEntity.notFound().build());
+  }
```

Comment:
How come we are using `ResponseEntity` here rather than throwing a `notFound` exception?

(My primary concern is that it is inconsistent with the pattern we use elsewhere.  If we are planning to change pattern, then I think it is fine but deserves a discussion as it would introduce technical debt in TNT, JIT and UI-Support)